stem/output.py

```diff
@@ -0,0 +1,219 @@
+import warnings
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List, Dict, Any
+from enum import Enum
+
+DEFAULT_GID_FLAGS = {
+    "WriteDeformedMeshFlag": "WriteUndeformed",
+    "WriteConditionsFlag": "WriteElementsOnly",
+    "GiDPostMode": "GiD_PostBinary",
+    "MultiFileFlag": "SingleFile",
+}
+
+
+class NodalOutput(Enum):
```

Comment:
can you also make an enum with gaussOutput with E.g. CAUCHY_STRESS_TENSOR, CAUCHY_STRESS_VECTOR etc....

my intention behind the Enum is to force users to use one of the Enum values. For example:

nodal_results1 = [NodalOutput.DISPLACEMENT, NodalOutput.TOTAL_DISPLACEMENT]